src/cls/IPM/Lifecycle/Base.cls

```diff
@@ -1656,24 +1678,22 @@ Method %MakeDeployed(ByRef pParams) As %Status
 				}
 			}
 		}
-		If $Data(tDeployClassArray) {
+		If ($Data(tDeployRtnArray) || $Data(tDeployClassArray)) {
 			Set ..HasDeployedResources = 1
-			If (tVerbose) {
-				Set tClass = ""
-				Write !,"The following classes are flagged for deployment:"
-				#; Write !,"The following classes "_$Select(tDev:"would be flagged",1:"are flagged")_" for deployment:"
-				For {
-					Set tClass = $Order(tDeployClassArray(tClass))
-					Quit:tClass=""
-					Write !,$Char(9),tClass
+			If $Data(tDeployClassArray) {
+				Set ..HasDeployedResources = 1
```

Comment:
@isc-shuliu  small thing but you set this property twice and also the $Data() check for the routine array went away for some reason. You can move the setting of .. properties + save to be the first thing in the outer if block and then do the separate if blocks for classes vs routines.